.github/environments.yaml

```diff
@@ -4,3 +4,19 @@ environments:
      - ../etc/base.yaml
      - ../etc/base-secrets.yaml
      - ../.github/install-all.yaml
+     - ../.github/secrets.yaml.gotmpl
+
+---
+
+helmDefaults:
+  atomic: {{ .Values.atomicInstall }}
+  kubeContext: {{ .Values.kubeContext }}
+  cleanupOnFail: true
+  wait: true
+  timeout: 600
```

Comment:
I’d prefer to keep this timeout variable per page helmfile, because waiting 10 minutes on a pod that should become healthy within 30 seconds is really excessive. For the atomic flag, perhaps we don’t need it anymore if the `needs` property works well?